gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,35 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed mostly through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create resources through Anypoint Platform _with the exception_ of Configuration and Service resources. 
```

Comment:
@seblaz What's are examples of a Service  and Configuration resources?

Reply:
We have examples in [these docs](https://docs.mulesoft.com/gateway/latest/flex-local-configuration-reference-guide). Although they are under "Manage Flex Gateway in Local Mode" they work for connected mode as well

Reply:
Hmm, I wonder if we need to document that in both places?